.github/workflows/build-test

```diff
@@ -33,9 +33,57 @@ rm -rf ${ARTIFACTSDIR} && mkdir ${ARTIFACTSDIR}
 
 python -m pip install --upgrade pip wheel
 
+function unsupported() {
+    case "$1" in
+        pytket-iqm)
+            # The iqm-client package is only available for 3.9.
+            [[ "${PYVER}" != "3.9" ]] && return
+            ;;
+        pytket-qiskit)
+            # Cannot install qiskit-aer on Windows with 3.10.
+            [[ "${PLAT}" == "Windows" && "${PYVER}" == "3.10" ]] && return
```

Comment:
Is this documented in an issue somewhere?

Reply:
No, but I will raise one.